readme.md

```diff
@@ -51,6 +51,13 @@ const execa = require('execa');
 	const {stdout} = await execa.shell('echo unicorns');
 	//=> 'unicorns'
 
+	// Cancelling a spawned process
+	const spawned = execa("echo unicorns");
+	spawned.catch( error => {
+		// This would be equal to the reason provided for canceling
+		console.log(error.message);
+	}); // To handle the CancelError
+	spawned.cancel("reason for canceling");
```

Comment:
To simplify the example I would personally call `cancel()` without arguments here, but it's up to you if you think it's better.

Reply:
what you are inferring seems better